### DISCUSSION TITLE
希望能增加mentions组件

### DISCUSSION THREAD

**Discussion by vvhr** [element-plus/element-plus#17695] [Ideas]
ant的mentions组件不仅用于@场景,还可以套用于需要在文本框插入占位符文本的场景,比如设计短信模板,设计公式等.属于一个比较实用的扩展文本框组件, 建议增加.

**Comment by warmthsea** [comment]
https://github.com/element-plus/element-plus/pull/17586

**Comment by elarbor** [comment]
还是[elementUI-vue3](https://element-plus.org/)用的惯，别的组件库我都用不惯😂

**Comment by kooriookami** [comment]
已添加
